cargo-dylint/tests/markdown_link_check.json

```
@@ -1,11 +1,23 @@
 {
   "httpHeaders": [
     { "urls": ["https://crates.io"], "headers": { "Accept": "text/html" } },
-    { "urls": ["https://github.com"], "headers": { "Accept": "text/html" } }
+    { 
+      "urls": ["https://github.com", "https://api.github.com"],
```

```suggestion
      "urls": ["https://github.com"],
```
I don't think any of the markdown files refer to https://api.github.com. Is this just for future-proofing?

---

Yes, you're right. I kept it before just in case if it was used in the future. Since none of the markdown files currently link to api.github.com, we don't need to include it. I've removed it as suggested.